psng/python/base.py

```diff
@@ -335,7 +335,7 @@ def set_zerro(self, s="XYZ", x=0.0, y=0.0, z=0.0):
             if "Z" in s:
                 tmpz = tmpz - z + self.halcomp["ps_offs_z"]
                 c += " Z%s" % tmpz
-            self.gcode(c)
+            self.gcode(c)                                 # Is there a real reason for not use the same system if self.gcode(c) == -1: for coding style ?
```

Comment:
This functional call probably should use `if self.gcode(c) == -1:`, returning -1 from this method on failure and 0 on success, just as the `def gcode` method does.